code/datums/spells/lordspells.dm

```diff
@@ -0,0 +1,90 @@
+// Necrolord
+/obj/effect/proc_holder/spell/targeted/trigger/soulflare
+	name = "Soulflare"
+	desc = "Deals high damage to an enemy in 3 different damage types, as well as paralyzing them for 1 seconds If it hits an enemy in critical condition, it instantly kills them and lowers the cooldown permanently, to a maximum of 6."
+	school = "transmutation"
+	charge_max = 300
+	clothes_req = 1
+	invocation = "NEKROSIS"
+	invocation_type = "shout"
+	message = "<span class='notice'>Your head feels like it's being burned as you fall to the ground!</span>"
+	cooldown_min = 300
+	level_max = 0 // no upgrades because it allows you to get 0 cooldown if you wait with upgrades.
+
+	starting_spells = list("/obj/effect/proc_holder/spell/targeted/inflict_handler/soulflare")
+
+	action_icon_state = "soulflare"
+
+	var/soulflarecm = 300 // debug purposes
+
+/obj/effect/proc_holder/spell/targeted/inflict_handler/soulflare
+	amt_paralysis = 1
+	amt_dam_fire = 15
+	amt_dam_brute = 15
+	amt_dam_oxy = 15
+	sound="sound/magic/Necrolord_Soulflare_Cast.ogg"
+
+/obj/effect/proc_holder/spell/targeted/inflict_handler/soulflare/cast(list/targets, mob/user = usr)
+	var/obj/effect/proc_holder/spell/targeted/trigger/soulflare/SF = locate(/obj/effect/proc_holder/spell/targeted/trigger/soulflare, user.mob_spell_list)
+	var/mob/living/carbon/target = targets[1]
+	if(target.health <= 0)
```

Comment:
I could definitely do something like this:
```
if(!target || !SF)
    return
```
before you even start on this line because if they don't have the spell or the target no longer exists then you'll want to make sure you handle it